src/modules/AccountSettings/SettingOg/AdvanceMode.tsx

```diff
@@ -57,7 +60,9 @@ export default function AdvanceMode({
   existingOg: CustomOg[] | null | undefined
 }) {
   const { toast } = useToast()
-  const [isSubmitting, setIsSubmitting] = useState<boolean>(false)
+  const [formState, setFormState] = useState<
+    'initializing' | 'ready' | 'submitting'
+  >('initializing')
```

Comment:
Most of the changes in this file is to improve the form UX:
1. Prevents submitting when no changes are made yet
2. Add `initializing` state.